src/index.js

```diff
@@ -70,1 +1,70 @@
+require('dotenv').config();
+
+const Discord = require('discord.js');

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/blackcatbot/blackcat/pullRequest?prid=8779465)
```suggestion
const Discord = require("discord.js");
```